extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -112,6 +113,10 @@ object PrivilegesBuilder {
         val cols = conditionList ++ aggCols
         buildQuery(a.child, privilegeObjects, projectionList, cols, spark)
 
+      case logicalRelation @ LogicalRelation(_: HadoopFsRelation, _, None, _) =>
```

Comment:
we don't need a new branch here

Reply:
If we call uriDescs for all cases of LogicalRelation, it may return two data containing both Table and Path, 
But we only want a path for cases that only have a path.

Reply:
> I don't quite follow

Update